go.mod

```diff
@@ -3,29 +3,40 @@ module github.com/trustwallet/blockatlas
 go 1.14
 
 require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
```

Comment:
We need to remove / indirect deps

Reply:
I reverted, can you check `go mod tidy` then?

Reply:
https://github.com/golang/go/issues/27428#issuecomment-417934130

Reply:
from the comment, we should include //indirect? always produce stable build

Reply:
If compiler cannot build it. We use Dockerfile for builds. There is no reason to add something if it works without it. Just my position about it, not a fact of course

Reply:
I don’t use docker when I develop

Reply:
Sure. But dev env != prod env